cli/src/main/java/org/aya/cli/utils/MainArgs.java

```diff
@@ -31,6 +31,8 @@ public static class CompileAction {
     public boolean isLibrary;
     @Option(names = {"--remake"}, description = "Treat input file as a library root and remake all")
     public boolean isRemake;
+    @Option(names = {"--check"}, description = "Treat input file as a library root and check all (no outputs will be saved to disk)")
```

Comment:
Maybe also called `--dry-run`?

Reply:
In Haskell I think there's a similar `--no-code`

Reply:
`--dry-run` means show files need processing, but do not process them. But we do tyck them.
`--no-code` looks funny and I'd like to accept it because I know [this repo](https://github.com/kelseyhightower/nocode) 🤣